### PR TITLE
English fix

### PR DESCRIPTION
**Pull request by EatVanced**

### CODE REVIEW

**Comment by nxzlem** [comment]
Does it follows the bebasit warnings as the repo [bebasid](https://github.com/bebasid/bebasid) did?

File: README.en.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <p align="center">
-    <b>For ISP that uses DPI, use <a href="https://github.com/bebasid/bebasit">bebasit</a> to bypass the DPI or turn on IPv6 if the ISP supports it.</b>
-    <br><sup>Signs if your ISP implemented DPI: HTTP redirected to<code><a href="http://lamanlabuh.aduankonten.id" target="_blank">lamanlabuh.aduankonten.id</a></code> or HTTPS responded with <code>ERR_CONNECTION_RESET</code>/<code>PR_CONNECT_RESET_ERROR</code></sup><br>
+    <b>For an ISP that uses DPI, use <a href="https://github.com/bebasid/bebasit">bebasit</a> to bypass the DPI or turn on IPv6 if the ISP supports it.</b>
```
Comment (nxzlem) [inline README.en.md:2]: This here

File: README.en.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
+    <br><sup>Signs that your ISP implemented DPI: HTTP redirected to<code><a href="http://lamanlabuh.aduankonten.id" target="_blank">lamanlabuh.aduankonten.id</a></code> or HTTPS responded with <code>ERR_CONNECTION_RESET</code>/<code>PR_CONNECT_RESET_ERROR</code></sup><br>
```
Comment (nxzlem) [inline README.en.md:3]: and right here

**Comment by EatVanced** [comment]
> Does it follows the bebasit warnings as the repo [bebasid](https://github.com/bebasid/bebasid) did?

oh yeah, it should be the same

**Review by VulcanSphere — APPROVED** [review]
Looks much better

**Comment by VulcanSphere** [comment]
> > Does it follows the bebasit warnings as the repo [bebasid](https://github.com/bebasid/bebasid) did?
> 
> oh yeah, it should be the same

Done, I have synchronised the warning to bebasid readme as well

**Pull request merged**